salt/modules/dockercompose.py

```diff
@@ -475,6 +488,7 @@ def create(path, docker_compose):
 
         salt myminion dockercompose.create /path/where/docker-compose/stored content
     """
+    # TODO: needs adjustment for python on whales
```

Comment:
This function (create) is not doing what `docker compose create` should be doing. This seems like an antipattern as every other compose function listed is a 1-1 cli command match.

as such i have not touched it yet as i feel we need community input

Reply:
Can you document this so users are aware when using the python on whales library.

Reply:
ive added a new function create_command. which actually runs the create command in docker-compose.

in the longterm the existing "create" command should probably become "create_file" or something similar.